pytest_mh/_private/plugin.py

```diff
@@ -862,6 +891,12 @@ def pytest_configure(config: pytest.Config):
         "the test is skipped if condition is not met",
     )
 
+    config.addinivalue_line(
+        "markers",
+        "preferred_topology(topology: str | None): mark test with a preferred topology."
```

Comment:
I would like to see:

`str | TopologyMark | list[str | TopologyMark]` and support these to make it more flexible

Reply:
We only expect a single value, so why accept more than one by adding lists?